packaging/linux/aur/PKGBUILD

```diff
@@ -20,8 +20,24 @@ source=("$pkgname::git+@GITHUB_CLONE_URL@#commit=@GITHUB_COMMIT@")
 sha256sums=('SKIP')
 
 prepare() {
-    cd "$pkgname"
-    git submodule update --recursive --init
+# Skip submodules that we don't want
+    if [[ $CARCH == "x86_64" ]]; then
+        git -c submodule."ffmpeg-macos-x86_64".update=none \
+            -c submodule."ffmpeg-windows-x86_64".update=none \
+            -c submodule."ffmpeg-linux-aarch64".update=none \
+            -c submodule."ffmpeg-macos-aarch64".update=none \
+            submodule update --recursive --init
+    elif [[ $CARCH == "aarch64" ]]; then
+        git -c submodule."ffmpeg-macos-x86_64".update=none \
+            -c submodule."ffmpeg-windows-x86_64".update=none \
+            -c submodule."ffmpeg-linux-x86_64".update=none \
+            -c submodule."ffmpeg-macos-aarch64".update=none \
+            submodule update --recursive --init
+
+# It's unlikely that someone could get this far on a system with an incorrect arch, but we should handle it anyway; pull everything
+    else
+        git submodule update --recursive --init
```

Comment:
Really no point in pulling the non linux ones... maybe set those to `update=none`?